### PR TITLE
Added a few slamdata libraries.

### PR DESCRIPTION
**Pull request by eskimor**
halogen-*
aff
affjax

### CODE REVIEW

File: libraries.json [hunk 1]
```diff
@@ -112,6 +120,18 @@
     "github": { "user": "purescript", "repo": "purescript-graphs" }
   },
   {
+    "bowerName": "purescript-halogen"
```
Comment (michaelficarra) [inline libraries.json]: Missing commas.

Comment (eskimor) [inline libraries.json]: Thanks! Already fixed, currently building locally to make sure everything is alright.

**Comment by michaelficarra** [comment]
:+1:

**Comment by eskimor** [comment]
It started building the database locally, without complaints - so, I assume: :+1:

**Comment by paf31** [comment]
Thanks!

**Pull request merged**